.gitignore

```diff
@@ -4,4 +4,5 @@
 lib
 node_modules
 reports
-*.log
\ No newline at end of file
+*.log
+**/.env*
```

Comment:
Please add test & build artifacts and any other generated files/artifacts to `.gitignore`. Right now, I'm seeing the `.nyc_output` directory being included, which I imagine should be ignored?